iocore/net/UnixUDPNet.cc

```diff
@@ -840,44 +843,68 @@ UDPNetHandler::mainNetEvent(int event, Event *e)
   (void)event;
   (void)e;
 
-  PollCont *pc = get_UDPPollCont(e->ethread);
+  UnixUDPConnection *uc;
+
+  // handle new UDP connection
+  SList(UnixUDPConnection, newconn_alink) ncq(newconn_list.popall());
+  while ((uc = ncq.pop())) {
+    if (uc->shouldDestroy()) {
+      uc->Release();
+    } else {
+      ink_assert(uc->mutex && uc->continuation);
+      open_list.enqueue(uc);
+    }
+  }
```

Comment:
The new udp connection is pushed into atomic list within `bindToThread`.
But no one popup the `new udp connection` from the atomic list.